outputs.tf

```
@@ -1,3 +1,13 @@
+output "cloudwatch_log_group_arn" {
+  description = "Map of CloudWatch log group type and arn when atleast one log type is enabled"
+  value       = { for log_type, log_configuration in var.cloudwatch_logging_configuration : log_type => aws_cloudwatch_log_group.default[log_type].arn if log_configuration.enabled }
+}
+
+output "cloudwatch_log_group_name" {
+  description = "Map of CloudWatch log group type and name when atleast one log type is enabled"
+  value       = { for log_type, log_configuration in var.cloudwatch_logging_configuration : log_type => element(split(":", aws_cloudwatch_log_group.default[log_type].arn), 6) if log_configuration.enabled }
```

```suggestion
  value       = { for log_type, log_configuration in var.cloudwatch_logging_configuration : log_type => aws_cloudwatch_log_group.default[log_type].name if log_configuration.enabled }
```

We should be able to access the [`name` field](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/cloudwatch_log_group#name) here; does this work for you?